docs/spark-writes.md

```diff
@@ -313,6 +313,28 @@ data.writeTo("prod.db.table")
     .createOrReplace()
 ```
 
+### Schema Merge
+
+While inserting or updating Iceberg is capable of resolving schema mismatch at runtime. If configured accordingly Iceberg will perform an automatic schema evolution as following:
```

Comment:
"configured accordingly" is a bit odd, "configured" alone is probably fine.

"as following" => "as follows"

Reply:
Minor nit, just added comma

If configured, Iceberg will perform an automatic schema evolution as follows: